src/main/java/org/broadinstitute/hellbender/tools/exome/ACSModeller.java

```diff
@@ -0,0 +1,150 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.tools.exome.allelefraction.AlleleFractionModeller;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.mcmc.PosteriorSummary;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents an ACS segmented model for copy ratio and allele fraction.
+ *
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ */
+public final class ACSModeller {
+    protected static final String INITIAL_SEG_FILE_TAG = "sim-0";
```

Comment:
If you could make these tags follow alphabetical order the results files would look nicer in a terminal.